src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+module.exports = function(options, undef) {
+  'use strict';
+
+  const TAU = 2 * Math.PI,
+        //TAU = options.PConstants.TAU,
+        lerp = function(start, stop, amt) {
+          return amt * (stop - start) + start;
+        },
+        sq = function(n) {
+          return n*n;
+        },
+        argsErr = function(mtd, len, min) {
```

Comment:
that works, although naming vars after what they do is usually better, so something involving "name" seems more to the point. `argsErr = function(fname, len, min)`?